mappings/net/minecraft/util/Identifier.mapping

```diff
@@ -1,30 +1,31 @@
 CLASS qs net/minecraft/util/Identifier
 	CLASS qs$a Serializer
-		METHOD deserialize (Lcom/google/gson/JsonElement;Ljava/lang/reflect/Type;Lcom/google/gson/JsonDeserializationContext;)Ljava/lang/Object;
-			ARG 1 functionJson
-			ARG 2 unused
-			ARG 3 context
 	FIELD a namespace Ljava/lang/String;
 	FIELD b path Ljava/lang/String;
-	FIELD c EXCEPTION_INVALID Lcom/mojang/brigadier/exceptions/SimpleCommandExceptionType;
-	METHOD <init> (Ljava/lang/String;)V
+	FIELD c COMMAND_EXCEPTION Lcom/mojang/brigadier/exceptions/SimpleCommandExceptionType;
+	METHOD <init> ([Ljava/lang/String;)V
 		ARG 1 id
-	METHOD <init> (Ljava/lang/String;Ljava/lang/String;)V
-		ARG 1 namespace
-		ARG 2 name
 	METHOD a getPath ()Ljava/lang/String;
-	METHOD a isValidChar (C)Z
-		ARG 0 chr
-	METHOD a parse (Lcom/mojang/brigadier/StringReader;)Lqs;
-	METHOD a create (Ljava/lang/String;)Lqs;
-		ARG 0 s
-	METHOD a createSplit (Ljava/lang/String;C)Lqs;
-		ARG 0 str
-		ARG 1 chr
+	METHOD a isCharValid (C)Z
+		ARG 0 c
+	METHOD a (I)Z
+		ARG 0 c
```

Comment:
What is this?

Reply:
A `char` lambda parameter

Reply:
I is integer, not char.

Reply:
It is a `char` represented as an `int` by an `IntConsumer` from the `IntStream` of `String#chars`: `String#chars().allMatch(c -> ...)`

Reply:
Is this a synthetic method generated for `METHOD a isCharValid (C)Z`?

Reply:
No, the two `IntConsumer` synthetic lambda methods are for `isNamespaceValid` and `isPathValid`

Reply:
We discussed it in discord. It's caused by String's `chars` method returning an `IntStream`.